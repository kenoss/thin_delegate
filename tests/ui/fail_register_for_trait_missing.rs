#[thin_delegate::register]
enum Hoge {
    A(String),
    B(char),
}

#[thin_delegate::derive_delegate]
impl ToString for Hoge {}

fn main() {}
