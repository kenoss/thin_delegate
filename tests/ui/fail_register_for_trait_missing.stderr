error: cannot find macro `__thin_delegate__feed_trait_def_of_Hello` in this scope
  --> tests/ui/fail_register_for_trait_missing.rs:13:6
   |
6  | #[thin_delegate::register]
   | -------------------------- similarly named macro `__thin_delegate__feed_structenum_def_of_Hoge` defined here
...
13 | impl Hello for Hoge {}
   |      ^^^^^ help: a macro with a similar name exists: `__thin_delegate__feed_structenum_def_of_Hoge`
